content/docs/user-guide/project-config/_index.md

```diff
@@ -1,13 +1,42 @@
+
+To build projects, you can use either Project Manager or **CMake**. You can find information on building projects in the [introductory tutorials](/docs/welcome-guide/create/) in the Getting Started Guide, and in the [Configure and Build](/docs/user-guide/build/configure-and-build) topic in the Build section of this User Guide.
+
+## Understanding `project.json` {#understanding-project-json}
+
+Each project contains a project manifest file named `project.json`. You can find this file in the project root directory. It stores important configuration properties for each project. It is created for you at the time of project creation.
+
+To make changes to any of the properties in this file, you can either edit it manually using a text editor, or specify individual properties to edit using the [O3DE CLI](./cli-reference) `edit-project-properties` command. Additionally, you can edit the `display_name` property in Project Manager.
+
+The following table describes each of the properties in `project.json`. The default value for many of the properties contains the name of the project given when the project was created. In the values below this is indicated by "<PROJECT_NAME>".
+
+| Property | Description | Default |
+| --- | --- | --- |
+| **project_name** | The name of the project. Used by the O3DE CLI `--project-name` parameter to identify the project. | "<PROJECT_NAME>" |
+| **origin** | A URL for the project, such as the repository URL or project website. This field is intended to be available for display and use by Project Manager. | "The primary repo for <PROJECT_NAME> goes here: i.e. http://www.mydomain.com" |
+| **license** | The license your project uses, and any copyright information you want to include. Consider providing the URL for the license. This field is intended to be available for display and use by Project Manager. | "What license <PROJECT_NAME> uses goes here: i.e. https://opensource.org/licenses/MIT" |
+| **display_name** | The display name of the project in Project Manager. | "<PROJECT_NAME>" |
+| **summary** | A short description of your project. This field is intended to be available for display and use by Project Manager. | "A short description of <PROJECT_NAME>." |
+| **canonical_tags** | A standard field in O3DE manifests used to identify what type of O3DE object this is. Examples: "Gem", "Project". Projects should use the "Project" tag. | [ "Project" ] |
+| **user_tags** | Any keyword tags that you want to associate with your project. Examples: "Physics", "Assets", "AWS". You can see examples of the standard set of tags used by Gems included in the Project Manager **Gem Catalog**. These tags are used by Project Manager for documentation, search, and filtering. | [ "<PROJECT_NAME>" ] |
+| **icon_path** | The path and filename of the project icon. This icon is used in **Project Manager**. It must be named `preview.png` and located in the project root directory. | "preview.png" <br> DO NOT CHANGE! |
```

Comment:
```suggestion
| **icon_path** | The path and filename of the project icon. Project Manager uses this icon. The file must be located in the project root directory, and the filename must be `preview.png`. <br><br> WARNING: Do not change the path or filename. | "preview.png" |
```
* Dropped the **bold** formatting from "Project Manager," as the tool is mentioned frequently prior to this description. (Use bold only on first occurrence.)
* Changed the ambiguous "it" ("...Project Manager. It....") to "the file."
* Edit for active voice ("used in Project Manager" > "Project Manager uses")
* Re-ordered the second sentence to bring the location/path directive to the front of the sentence. This is more consistent with the property name ("icon_path") and the earlier wording order ("path and filename").
* Reworded the warning from "DO NOT CHANGE!" to a more formal "WARNING" note/alert.
  * Consider moving this into the description. IMO, it gets lost in the "Default" column, visually blurring with the all-caps "PROJECT_NAME".